tests/behat/features/file-download.d7.feature

```diff
@@ -0,0 +1,36 @@
+@d7
+Feature: Check that FileDownloadTrait for D7 works
+
+  Background:
+    Given I am logged in as a user with the "administrator" role
+    And article content:
+      | title                | field_file                                                   |
+      | [TEST] document page | file;example_files/example_document.txt;example_document.txt |
```

Comment:
please update these to use `fixture` files as below `example_document.pdf` etc.  If you need a `.txt` file - please add it to `tests/fixtures` directory.

`tests/fixtures` is automatically discovered by DrupalExtension